pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -301,6 +306,11 @@ func (config *V2RegistriesConf) postProcessRegistries() error {
 			}
 		}
 
+		// ignore the credHelper if it is set for inner namespace
+		if reg.CredentialHelper != "" && strings.Contains(reg.Prefix, "/") {
```

Comment:
@QiWang19, can you elaborate _why_ we have that restriction?

The `strings.Contains(reg.Prefix, "/")` check is an overestimation since a registry _may_ be running on domain.com/registry.  I haven't seen it yet but it's technically possible so I want to make sure to fully understand the restriction.  We may need to mention the reasoning also in the man page.